src/components/dev-popover/dev-popover.module.css

```diff
@@ -5,15 +5,12 @@
 .button {
```

Comment:
These changes were made to make the focus ring for `HeaderSearchInput` nicer. This component is used in `SidebarBackToLink` as well so that will be affected, but I'm not concerned about it because I will be working on `Sidebar` over the next two weeks.